pkg/addons/data/ConfigMap/openshift-azure-monitoring/prometheus-forwarder-config.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+data:
+  config.yaml: |  
+    rule_files:
+    - '*.rules'
+    scrape_configs:
+    - job_name: cluster-prometheus
+      scrape_interval: 30s
+      bearer_token_file: "/run/secrets/kubernetes.io/serviceaccount/token"
+      scheme: https
+      tls_config:
+        ca_file: "/config/ca.crt"
+      params:
+        'match[]':
+          - '{__name__=~".+"}'
```

Comment:
We should remove rules from config above and just control metrics we pass here as this ipacts our prometheus performance directly